astroid/protocols.py

```diff
@@ -672,7 +672,7 @@ def named_expr_assigned_stmts(
 
 
 @decorators.yes_if_nothing_inferred
-def starred_assigned_stmts(
+def starred_assigned_stmts(  # noqa: C901
```

Comment:
Should we maybe just set the limit a little higher? We seem to be adding a lot of diables.

Reply:
Agreed.  Let's `noqa` the three or four most complex functions and then set max-complexity above the other.

Reply:
It means we can introduce some new functions that will be too complex in the future. There isn't that much complex functions to noqa (and I did noqa them all already). This is the reasoning we had when we did not filter the whole protocol file but added a noqa instead.